internal/docker/client.go

```diff
@@ -149,7 +149,9 @@ func (d *Client) FindContainer(id string) (Container, error) {
 	return container, nil
 }
 
-func (d *Client) ContainerActions(action string, id string) error {
+func (d *Client) ContainerActions(action string, actionContainer Container) error {
```

Comment:
Was there a reason you had to do this? I was looking at the other functions and all of them take `id`. So I think we should be consistent here and use `id`.  It also should make testing easier.

Reply:
Yeah, but I actually thought that I could use the result of `FindContainer`, since it would be an added check, that we are actually doing the actions on the intended container.

Reply:
But anyways, I could use the `container.ID` property. So I would better take your suggestions :)

Reply:
In this situation it might a premature optimization which breaks consistency and simplicity. So until we really need to pass the whole container as object, id should be enough.